hooks/post_gen_project.py

```diff
@@ -57,11 +57,12 @@ def get_random_secret_key():
     return "".join(secrets.choice(chars) for i in range(50))
 
 
-def remove_web_client_files(client):
-    rmtree(join(web_clients_path, client))
+web_clients_path = "clients/web"
+mobile_clients_path = "clients/mobile"
 
 
-web_clients_path = "clients/web"
+def remove_web_client_files(client):
+    rmtree(join(web_clients_path, client))
```

Comment:
I've noticed that the `clients` directory is now left behind (and empty) after generating a new project